www/assets/js/databrowser.js

```diff
@@ -163,7 +165,6 @@ browser = simply.app({
             }
         },
         '/': function(params) {
```

Comment:
de '/' route staat er nu 2 keer in, de 2e moet weg. Maar er is sowieso een issue, de / route triggert altijd, dus de routes daarna nooit. Dat moet aangepakt.